src/transformers/models/opt/modeling_opt.py

```diff
@@ -129,8 +129,8 @@ def _handle_deprecated_argument(config_arg_name, config, fn_arg_name, kwargs):
             val = None
             if fn_arg_name in kwargs:
                 logging.warning(
-                    "Passing in {} to {self.__class__.__name__} is deprecated and won't be supported from v4.38."
-                    " Please set it in the config instead"
+                    "Passing in {fn_arg_name} to {self.__class__.__name__} is deprecated and won't be supported from "
+                    "v4.39. Please set it in the config instead"
```

Comment:
This warning was not being print correctly (missing `fn_arg_name`), so I've pushed the deprecation cycle by 1 minor version